### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by nostadt**
I have moved some hard-coded translations from template files into XLF files as well as added the German translation for the backend module.

### CODE REVIEW

File: Resources/Private/Language/de.locallang.xlf [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+				<source>Select file</source>
+				<target>Datei auswählen</target>
+			</trans-unit>
+			<trans-unit id="choose_language">
+				<source>Select language</source>
+				<target>Sprache auswählen</target>
+			</trans-unit>
+			<trans-unit id="save">
+				<source>Save</source>
+				<target>speichern</target>
```
Comment (rrrapha) [inline Resources/Private/Language/de.locallang.xlf]:
```suggestion
				<target>Speichern</target>
```

File: Resources/Private/Language/de.locallang.xlf [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+				<source>Create file</source>
+				<target>Datei erstellen</target>
+			</trans-unit>
+			<trans-unit id="load_labels_from_csv_file">
+				<source>Load labels from CSV file</source>
+				<target>Übersetzungen aus CSV-Datei laden</target>
+			</trans-unit>
+			<trans-unit id="add">
+				<source>Add</source>
+				<target>neue Zeile</target>
```
Comment (rrrapha) [inline Resources/Private/Language/de.locallang.xlf]:
```suggestion
				<target>Neue Zeile</target>
```

File: Resources/Private/Templates/Module/Search.html [hunk 3]
```diff
@@ -7,7 +7,7 @@ <h1>{f:translate(key:'search')}</h1>
 		<f:form action="search" controller="Module" class="form-group">
 			<div class="form-group">
 				<div class="input-group">
-					<f:form.textfield name="word" value="{word}" class="form-control" placeholder="Label"/>
+					<f:form.textfield name="word" value="{word}" class="form-control" placeholder="{f:translate(key:'key')}"/>
```
Comment (rrrapha) [inline Resources/Private/Templates/Module/Search.html:10]: The should be the the word "Label", or something like `{f:translate(key:'label')}`.
```suggestion
					<f:form.textfield name="word" value="{word}" class="form-control" placeholder="Label"/>
```

**Review by rrrapha — CHANGES_REQUESTED** [review]
Thanks!

File: Resources/Private/Templates/Module/Search.html (hunk 3 above)
Comment (nostadt) [reply Resources/Private/Templates/Module/Search.html:10]: The search is done within the translations but not the key, correct? I agree, `key` is the wrong text then. Perhaps sth. like "translation" (en), "Übersetzung" (de)

Comment (rrrapha) [reply Resources/Private/Templates/Module/Search.html:10]: I'd prefer "Label" for both languages, or maybe just "Text"..

Comment (nostadt) [reply Resources/Private/Templates/Module/Search.html:10]: Hmm. I prefer text. If you find time to adjust today, feel free to do so. I may find time somewhen this weekend.

**Pull request merged**